### PR TITLE
[DefaultLimitNOFILE=50000] setting modification in Ubuntu16/18 images

### PR DESCRIPTION
**Pull request by andy-mishechkin**
[DefaultLimitNOFILE=50000] setting has been added to /etc/systemd/system.d for increasing the max number of open files in non-interactive sessions.

### CODE REVIEW

File: images/linux/ubuntu1604.json [hunk 1]
```diff
@@ -77,7 +77,8 @@
                 "systemctl disable apt-daily-upgrade.service",
                 "echo '* soft nofile 50000 \n* hard nofile 50000' >> /etc/security/limits.conf",
                 "echo 'session required pam_limits.so' >> /etc/pam.d/common-session",
-                "echo 'session required pam_limits.so' >> /etc/pam.d/common-session-noninteractive"
+                "echo 'session required pam_limits.so' >> /etc/pam.d/common-session-noninteractive",
+                "echo 'DefaultLimitNOFILE=50000' >> /etc/systemd/system.conf"
```
Comment (sergey-akhalkov) [inline images/linux/ubuntu1604.json]: Just wondering, why the limit is specified to 50000 (no more no less) ?

Comment (miketimofeev) [reply images/linux/ubuntu1604.json]: @sergey-akhalkov we don't know either looks like some random value that is definitely enough(max value is 65536)

Comment (andy-mishechkin) [reply images/linux/ubuntu1604.json]: I just repeat the previous limit value (in ["echo '* soft nofile 50000 \n* hard nofile 50000' >> /etc/security/limits.conf",])

Comment (al-cheb) [reply images/linux/ubuntu1604.json]: Seems like the PR contains a lot of non relative files

**Pull request closed without merging**